web-ide/web_ide.py

```diff
@@ -213,13 +213,19 @@ async def serve(q: Q):
 
     if q.args.export:
         await export(q)
-    elif q.args.code:
+    elif q.args.dropdown == 'code':
+        q.user.view = "code"
+        q.page['header'].items[3].dropdown.value = 'code'
```

Comment:
Can we do something more general like
```suggestion
    elif q.args.dropdown:
        q.user.view = q.args.dropdown
        q.page['header'].items[3].dropdown.value = q.args.dropdown
```

Would require changing elifs to ifs though.